### PR TITLE
Tools now fit in all versions of the military belt

### PR DESCRIPTION
**Pull request by Daylight2**
## What Does This PR Do
Makes tools fit in all versions of the military belt

## Why It's Good For The Game
Right now, military belts are just plastic bags with the added bonus of getting you arrested by security for c class. This makes them an actual upgrade, and also means that nukies can put the crowbar they start with into their belt.

## Testing
Put a bunch of tools in a bunch of belts

## Changelog
:cl:
tweak: Military belts can now hold tools
/:cl:

### CODE REVIEW

**Comment by Fordoxia** [comment]
Finally, a reason to not throw away military belts when I am an engineer.

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -378,6 +378,15 @@
 	max_w_class = WEIGHT_CLASS_SMALL
 	max_combined_w_class = 18
 	resistance_flags = FIRE_PROOF
+	use_item_overlays = TRUE // So it will still show tools in it in case sec get lazy and just glance at it.
```
Comment (Contrabang) [inline code/game/objects/items/weapons/storage/belt.dm:381]: This comment applies less to the normal military belt, can you change this comment?

File: code/game/objects/items/weapons/storage/belt.dm [hunk 2]
```diff
@@ -378,6 +378,15 @@
...
 	resistance_flags = FIRE_PROOF
+	use_item_overlays = TRUE // Will show the tools on the sprite
+	w_class_override = list(
+		/obj/item/crowbar,
+		/obj/item/screwdriver,
+		/obj/item/weldingtool,
+		/obj/item/wirecutters,
+		/obj/item/wrench,
+		/obj/item/multitool
+		)
```
Comment (Contrabang) [inline code/game/objects/items/weapons/storage/belt.dm:389]: One last thing for consistency
```suggestion
	)
```

**Review by Burzah — APPROVED** [review]

**Comment by S34NW** [comment]
Please merge `upstream/master` into your branch to fix CI failing due to external factors.

**Review by S34NW — APPROVED** [review]

**Pull request merged**